src/qt/walletview.cpp

```diff
@@ -386,7 +397,11 @@ void WalletView::gotoSigmaPage()
 
 void WalletView::gotoZc2SigmaPage()
 {
-    setCurrentWidget(zc2SigmaPage);
+    if (pwalletMain->IsHDSeedAvailable()) {
+        setCurrentWidget(zc2SigmaPage);
+    } else {
+        setCurrentWidget(sigmaPage);
```

Comment:
Better to provide comment here so the other people will not confuse why if no HD seed we will switch to sigma page instead of remint page.